types/mertasan__tailwindcss-variables/tsconfig.json

```diff
@@ -0,0 +1,28 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6",
+            "ES2018.Promise"
+        ],
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictFunctionTypes": true,
+        "strictNullChecks": true,
+        "esModuleInterop": true,
```

Comment:
```suggestion
```
why do you need this?
https://github.com/DefinitelyTyped/DefinitelyTyped#esmoduleinteropallowsyntheticdefaultimports

Reply:
@peterblazejewicz tailwindcss exports their types as `export = `

Reply:
I've added a comment, just update your imports, people using this DT module could use `esModuleInterop` freely:
```ts
import * as variablesPlugin from '@mertasan/tailwindcss-variables';
import * as  variablesApi from '@mertasan/tailwindcss-variables/api';
import colorVariable = require('@mertasan/tailwindcss-variables/colorVariable');
import * as plugin from 'tailwindcss/plugin';
import type { Config } from 'tailwindcss';
```